test/test_utils/security.py

```diff
@@ -19,7 +19,7 @@
 @dataclass
```

Comment:
ran autoformatter on this file

Reply:
What's preventing us from running a formatter over the entire project? Let's do that!

Reply:
Best done in a formatter-only PR which also adds in linter/formatting tests to ensure that standards are met going forward. Whole repo is out of scope for this PR.

Reply:
Agreed. Definitely out of scope for this PR.